test/test_tornado.py

```diff
@@ -52,7 +52,7 @@ def _get_response(self, url: str, benchmark: Callable) -> BaseServer.Response:
     def assert_404(self, url: str):
         with pytest.raises(HTTPClientError) as e:
             self._get_response(url, lambda f: f())
-            assert e.value.code == 404
+        assert e.value.code == 404
```

Comment:
`.value` must in fact be called _outside_ the context manager (https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions)

Otherwise, the `assert` does not raise an error if the condition is not true :shrug: